src/unity/schemas/Me.yml

```
@@ -37,8 +37,11 @@ properties:
           - read-only
           - read-write
     description: role permission whether this operator is "read-only" | "read-write"
+  accountCreatedAt:
+    type: string
+    description: a timestamp that indicates when the account was created
```

```suggestion
    description: an ISO8601 timestamp that indicates when the account was created
```

If you take this suggestion, then can you also add the same change to `paygCreditStartDate` below?